Tests/AmplitudeTests/Utilities/EventPipelineTests.swift

```diff
@@ -10,7 +10,7 @@ import XCTest
 @testable import AmplitudeSwift
 
 final class EventPipelineTests: XCTestCase {
-    private static let FLUSH_INTERVAL_SECONDS = 1.0
+    private static let FLUSH_INTERVAL_SECONDS = 300.0
```

Comment:
Are there no tests that depend on this being 1 second?

Reply:
There are none, but this isn't the proper fix - I can't reproduce the failures locally so I'm using this PR to debug.